include_core/AtomicSupport.hpp

```diff
@@ -292,24 +287,8 @@ class VM_AtomicSupport
 		__isync();
 #elif defined(_MSC_VER)
 		_ReadWriteBarrier();
-#if defined(J9HAMMER)
-		__faststorefence();
-#else /* J9HAMMER */
-		__asm { lock or dword ptr [esp],0 }
-#endif /* J9HAMMER */
-		_ReadWriteBarrier();
 #elif defined(__GNUC__)
-#if defined(J9X86)
-		asm volatile("lock orl $0x0,(%%esp)" ::: "memory");
-		/* TODO investigate whether or not we should call this instead
-		asm volatile("lfence":::"memory");
-		*/
-#elif defined(J9HAMMER)
-		asm volatile("lock orq $0x0,(%%rsp)" ::: "memory");
-		/* TODO investigate whether or not we should call this instead
-		asm volatile("lfence":::"memory");
-		*/
-#elif defined(ARM) /* defined(J9HAMMER) */
+#if defined(ARM)
```

Comment:
The comment on line 297 needs to be updated as well

Reply:
Comments addressed.